templates/images-template.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Image
+metadata:
+  name: node-image
+spec:
+  managementPolicy: managed
+  resource:
+    name: ubuntu-2404-kube-v1.31.2
+    content:
+      diskFormat: qcow2
+      download:
+        url: ${NODE_IMAGE_URL:="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/ubuntu/2024-11-21/ubuntu-2404-kube-v1.31.2.img"}
```

Comment:
I want to point this out to make sure it is ok. I have here linked to the image we (are about to) use in our e2e tests. I assume that no one will use this for any kind of serious work, but I hope it is ok for developers to use it. It is anyway not a secret where we pull it from.
@mdbooth do you know if it will generate any kind of costs for the project?

Reply:
I believe it does generate costs for the project, and I *think* we're discouraged from doing this. I wonder if it's worth asking in... #sig-k8s-infra? Perhaps it's ok, or perhaps there's a better place to host these things.

Reply:
Can't we do flatcar images with sysext completely from their public repos? If so, I'd be ok making flatcar our default repo here.

cc @tormath1